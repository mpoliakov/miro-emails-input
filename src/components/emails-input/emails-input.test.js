import EmailsInput from './emails-input';

describe('Testing EmailsInput component...', () => {
  it('EmailsInput component snapshot', () => {
    const container = document.createElement('div');
    const emailInputComponent = new EmailsInput(container);
    expect(emailInputComponent.getElement()).toMatchSnapshot();
  });
});
