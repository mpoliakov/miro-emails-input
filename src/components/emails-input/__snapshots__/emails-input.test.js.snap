// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing EmailsInput component... EmailsInput component snapshot 1`] = `
<div
  class="emails-input"
>
  
      
  <div
    class="emails-input__container"
  >
    <div
      class="emails-input__email email"
    >
      ann@miro.com
        
      <button
        class="email__btn-delete"
        title="Delete email"
      >
        ✖
      </button>
      
    
    </div>
    <div
      class="emails-input__email email email--invalid"
    >
      invalid.email
        
      <button
        class="email__btn-delete"
        title="Delete email"
      >
        ✖
      </button>
      
    
    </div>
    <div
      class="emails-input__email email"
    >
      john@miro.com
        
      <button
        class="email__btn-delete"
        title="Delete email"
      >
        ✖
      </button>
      
    
    </div>
    <div
      class="emails-input__email email"
    >
      mpoliakov@gmail.com
        
      <button
        class="email__btn-delete"
        title="Delete email"
      >
        ✖
      </button>
      
    
    </div>
  </div>
  <input
    class="emails-input__input"
    placeholder="add more people..."
    type="email"
  />
  
    
</div>
`;
